shell/platform/android/test/io/flutter/plugin/platform/PlatformViewsControllerTest.java

```diff
@@ -129,7 +129,7 @@ public void itCancelsOldPresentationOnResize() {
   @Test
   @Config(shadows = {ShadowFlutterJNI.class, ShadowPlatformTaskQueue.class})
   public void virtualDisplay_handlesResizeResponseWithoutContext() {
-    final int platformViewId = 0;
+    final long platformViewId = 0L;
```

Comment:
Consider making one of these values actually large enough to require a long.

Reply:
done.